test/e2e/scaffold/scaffold.go

```diff
@@ -407,3 +417,16 @@ func waitExponentialBackoff(condFunc func() (bool, error)) error {
 	}
 	return wait.ExponentialBackoff(backoff, condFunc)
 }
+
+// generateWebhookCert generates signed certs of webhook and create the corresponding secret by running a script.
+func generateWebhookCert(ns string) error {
+	commandTemplate := `../../utils/webhook-create-signed-cert.sh`
```

Comment:
Why not prepare static files and put them into `testdata` dir?

Reply:
good idea, I didn't notice the `testdata` dir